bin/waagent2.0

```diff
@@ -3300,6 +3300,87 @@ class Certificates(object):
             filename = str(index) + ".prv"
         return self
 
+
+class RemoteAccess(object):
```

Comment:
waagent2.0 is deprecated, please revert changes to this file.

Reply:
done